spec/lib/pg_search/multisearchable_spec.rb

```diff
@@ -69,8 +73,13 @@
         context "when the document is missing" do
           before { record.pg_search_document = nil }
 
-          describe "saving the record" do
+          describe "saving the record without changing any in it" do
+            it "should not create a PgSearch::Document record" do
```

Comment:
This is actually wrong, anyway, I think. If the PgSearch document is missing, we should still create it even if no data changed.